docs/type.rst

```diff
@@ -5,11 +5,47 @@ Supported types
 ---------------------------------------
 Currently, supported basic types in Taichi are
 
+- int8 ``ti.i8``
+- int16 ``ti.i16``
 - int32 ``ti.i32``
 - int64 ``ti.i64``
+- uint8 ``ti.u8``
+- uint16 ``ti.u16``
+- uint32 ``ti.u32``
+- uint64 ``ti.u64``
 - float32 ``ti.f32``
 - float64 ``ti.f64``
 
+.. note::
+    Supported types on each backends:
+
+    +------+-----------+-----------+---------+
+    | type | LLVM      | OpenGL    | Metal   |
+    +======+===========+===========+=========+
+    | i8   |    OK     |   MISS    |   OK    |
+    +------+-----------+-----------+---------+
+    | i16  |    OK     |   MISS    |   OK    |
+    +------+-----------+-----------+---------+
+    | i32  |    OK     |    OK     |   OK    |
+    +------+-----------+-----------+---------+
+    | i64  |    OK     |   EXT     |  MISS   |
+    +------+-----------+-----------+---------+
+    | u8   |    OK     |   MISS    |   OK    |
+    +------+-----------+-----------+---------+
+    | u16  |    OK     |   MISS    |   OK    |
+    +------+-----------+-----------+---------+
+    | u32  |    OK     |   MISS    |   OK    |
+    +------+-----------+-----------+---------+
+    | u64  |    OK     |   MISS    |  MISS   |
+    +------+-----------+-----------+---------+
+    | f32  |    OK     |    OK     |   OK    |
+    +------+-----------+-----------+---------+
+    | f64  |    OK     |   EXT     |  MISS   |
+    +------+-----------+-----------+---------+
```

Comment:
Please verify if I told correct on your backends @yuanming-hu @k-ye. Thanks!

Reply:
LLVM looks good! Thanks for adding the table. It's very helpful.